docs/clipboard-unsanitized/HOWTO.md

```diff
@@ -0,0 +1,40 @@
+# How to use the Async Clipboard API to read and write unsanitized HTML
+**Last updated: March, 2023**
+
+Reading and writing unsanitized HTML to and from the clipboard is currently available in Chromium-based browsers in 113 and later behind the flag `ClipboardUnsanitizedContent`.
+1. Download Microsoft Edge ([Canary Channel](https://www.microsoftedgeinsider.com/en-us/download/canary)).
```

Comment:
Should we mention that similar process can be followed to test this on Chrome Canary?

Reply:
Done.